### PR TITLE
Survey accounts empty instruments fix

### PR DESCRIPTION
**Pull request by intralizee**
This pull request `Fix Warning: Undefined variable: instrument in Utility.class.inc`.

See also: `Task #14375`

### CODE REVIEW

File: php/libraries/Utility.class.inc [hunk 1]
```diff
@@ -725,11 +725,12 @@ class Utility
      * This should return an array in a format suitable for addSelect() from
      * NDB_Page
      *
-     * @return associative array of test_names in a Test_Name => "Full Name"
+     * @return array $instruments associative array of test_names in a Test_Name => "Full Name"
```
Comment (driusan) [inline php/libraries/Utility.class.inc]: I don't think the variable name is meaningful in the context of defining the return, it's a meaningless internal detail, but other than that +1

**Comment by zaliqarosli** [comment]
linking this https://redmine.cbrain.mcgill.ca/issues/14375 for accessibility

**Comment by intralizee** [comment]
Seems like a PR was merged that solves this bug and so closing it.

**Comment by ridz1208** [comment]
This PR seems to only be moving a line downwards... approving anyways

**Comment by driusan** [comment]
I don't see how this would fix any warning. It seems to just be moving a variable declaration from the top of the function to slightly further down?

**Comment by intralizee** [comment]
@driusan If you checkout the bugfix branch you will notice the variable not initiated and I must have incorrectly rebased before which is why I just moved the variable so git would notice a file change.

**Pull request closed without merging**